File: LAB/Lab_1/Z1.ml
let fiddle4 (a, b, c, d) = 
  let third = c -. b in
  (d, a, third)

let printTuple (a, b, c) =
  Printf.printf "(%.2f, %.2f, %.2f)\n" a b c

let result = fiddle4(1.3, 2.0, 3.1, 4.2);;
printTuple result;;

File: LAB/Lab_1/Z1.scala
def fiddle4(tuple: (Double, Double, Double, Double)): (Double, Double, Double) = {
    (tuple._4, tuple._1, tuple._3 - tuple._2);
}

@main def main() = {
    val tuple = (1.3, 2.0, 3.1, 4.2);
    println(fiddle4(tuple));
}